### PR TITLE
Refactor subscriptions to async pipes

### PR DESCRIPTION
**Pull request by GoodDayForSurf**
- refactor

### CODE REVIEW

File: packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html [hunk 1]
```diff
@@ -10,25 +10,25 @@
...
-      <dxo-size [height]="90"></dxo-size>
-
-      <dxo-loading-indicator [show]="true"></dxo-loading-indicator>
-
-      <dxo-chart>
-        <dxi-series type="line" argumentField="date" valueField="total">
-        </dxi-series>
-      </dxo-chart>
-    </dx-range-selector>
+      <dx-range-selector [dataSource]="sales" [(value)]="visualRange" (onValueChanged)="onRangeChanged($event)">
```
Comment (ivanblinov2k17) [inline packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html:13]: Are these spaces essential?

File: packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html [hunk 2]
```diff
@@ -77,11 +77,11 @@
...
     </card-analytics>
 
-    <card-analytics titleText="Sales Performance" className="sales">
+    <card-analytics titleText="Sales Performance">
       <div filter class="sales-filter" >
-      <dx-drop-down-button stylingMode="text" [useSelectMode]="true"
-        [items]="groupByPeriods" [selectedItemKey]="groupByPeriods[1]" (onSelectionChanged)="selectionChange($event)">
-      </dx-drop-down-button>
+        <dx-drop-down-button stylingMode="text" [useSelectMode]="true"
+                             [items]="groupByPeriods" [selectedItemKey]="groupByPeriods[1]" (onSelectionChanged)="selectionChange($event)">
```
Comment (ivanblinov2k17) [inline packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html:83]: whats this tab for?

File: packages/angular/src/app/pages/planning-task-details/planning-task-details.component.ts [hunk 3]
```diff
@@ -51,10 +48,6 @@ export class PlanningTaskDetailsComponent implements OnInit, OnDestroy {
     this.loadData();
   }
 
-  ngOnDestroy(): void {
```
Comment (ivanblinov2k17) [inline packages/angular/src/app/pages/planning-task-details/planning-task-details.component.ts:54]: why this unsubscribe is removed? does it happen automatically or in some other place?

Comment (GoodDayForSurf) [reply packages/angular/src/app/pages/planning-task-details/planning-task-details.component.ts:54]: All http subscriptions always сomplete therefore no need to unsubscribe

File: packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html (hunk 2 above)
Comment (GoodDayForSurf) [reply packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html:83]: it's github's formatting, it's not in the source

File: packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html (hunk 1 above)
Comment (GoodDayForSurf) [reply packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html:13]: as child in parent

**Review by ivanblinov2k17 — APPROVED** [review]

**Review by groshenkovamarina — APPROVED** [review]

**Pull request merged**